stable/burrow/Chart.yaml

```diff
@@ -1,7 +1,8 @@
 name: burrow
-version: 0.4.2
+version: 0.4.3
 appVersion: 0.17.1
 description: Burrow is a permissionable smart contract machine
+home: https://burrow.com/
```

Comment:
This site is about a luxury couch...

Reply:
Hi @timstoop , I modified home key with "https://github.com/hyperledger/burrow"
This is from the README.md of burrow.